src/components/molecules/VisualisationPreview.tsx

```diff
@@ -21,6 +23,16 @@ type VisualisationPreviewProps = {
 const VisualisationPreview: React.FC<VisualisationPreviewProps> = ({ metadata, selectedVisualisation }) => {
     const [paragraph, setParagraph] = useState<string>();
     const [size, setSize] = useState<DashboardItemSize>(DashboardItemSize.LARGE);
+    const dashboardItems = useReactiveVar(dashboardItemsVar);
+
+    useEffect(() => {
+        if (!dashboardItems) return;
+        const item = dashboardItems.find((el) => el.id === metadata.id);
+        if (item?.size) {
+            setSize(item.size);
+        }
+        setParagraph(item?.paragraph);
+    }, []);
```

Comment:
Shouldn't dashboardItems be inside the []?

Reply:
yup, thanks 🙏